src/qibolab/channel_configs.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass
+from typing import Optional, Union
+
+from .execution_parameters import AcquisitionType
+
+"""Common configuration for various channels."""
+
+
+@dataclass(frozen=True)
+class DCChannelConfig:
```

Comment:
This is just a matter of style and opinions, but given the file scope, and how they will be used, I'd even go for more minimal names
```suggestion
class DC:
```

Reply:
Yeah, well I don't have any strong preference. If majority voted for shorter names, then it is the way to go. 

I originally preferred to keep them with full names, because these configs are going to be exposed to end users, and I aimed at naming that keeps their code nicer and more readable. They can of course to `from qibocal import channel_config`, and then use the various classes as `channel_config.DC`.

Reply:
> They can of course do `from qibocal import channel_config`, and then use the various classes as `channel_config.DC`

Indeed, the idea is that if you want the scope, you have it. It's just optional.

Reply:
So after all the latest changes these things currently look like: `DcConfig`, `IqConfig`, etc. Since the concept of channel is replaced with more general concept of component, which channel is a subset of, there are things like `OscillatorConfig`, which I don't think should be reduced to `Oscillator`, so keeping the word config in the name makes code (ours, and user's) more readable.

Reply:
Indeed, now I support the opposite: use `Iq` for the `IqChannel`.